src/scenic/simulators/carla/blueprints.scenic

```diff
@@ -7,15 +7,15 @@ carModels = {# for speed = 10
       'vehicle.audi.etron', # K_P=0.3, K_D=0.2, K_I=0
       'vehicle.audi.tt',
       'vehicle.bmw.grandtourer',
-      # 'vehicle.bmw.isetta', # current PID controller not work 
+      'vehicle.bmw.isetta', # current PID controller not work 
```

Comment:
Probably should remove the comment if it's not accurate anymore. Ditto for "wrangler_rubicon" below.

Reply:
Done

Reply:
Comments are still there - maybe another commit is missing?